warp-core/src/main/scala/com/workday/warp/arbiters/ArbiterLike.scala

```diff
@@ -80,7 +80,8 @@ trait ArbiterLike extends PersistenceAware with CanReadHistory {
     * @return a wrapped error with a useful message, or None if the measured test passed its requirement.
     */
   final def voteWithSpikeFilter[T: TestExecutionRowLikeType](ballot: Ballot, testExecution: T): Option[Throwable] = {
-    val (spikeFilterEnabled, alertOnNth) = spikeFilterSettings(testExecution.idTestDefinition)
+    val methodSignature: String = this.persistenceUtils.getMethodSignature(testExecution)
```

Comment:
```suggestion
    val methodSignature: String =  ballot.testId.id
```

Just one suggestion here to avoid a db query